mkdocs.yml

```diff
@@ -34,6 +34,14 @@ theme:
       toggle:
         icon: material/brightness-4
         name: Switch to light mode
+copyright: Copyright &copy 2023 dynamiqs
+extra:
+  generator: false
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/dynamiqs/dynamiqs
+    - icon: fontawesome/brands/slack
+      link: https://dynamiqs-org.slack.com/
```

Comment:
We should put the invite link (it redirects automatically if you signed up on the Slack)

https://join.slack.com/t/dynamiqs-org/shared_invite/zt-1z4mw08mo-qDLoNx19JBRtKzXlmlFYLA